saber/dvid_access/dvid_access.py

```diff
@@ -0,0 +1,188 @@
+# Copyright 2019 The Johns Hopkins University Applied Physics Laboratory
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/usr/bin/env python
+
+import argparse
+import configparser
+import tempfile
+import boto3
+from intern.remote.dvid import DVIDRemote
+from intern.resource.dvid import DataInstanceResource
+import numpy as np
+from requests import HTTPError
+
+def DVID_pull_cutout(args):
+    rmt = DVIDRemote({
+        "protocol": "http",
+        "host": args.host
+    })
+
+    # Create or get a channel to write to
+    instance_setup = DataInstanceResource(
+        UUID=args.uuid, name=args.data_instance, type=args.type, alias=args.alias, datatype=args.datatype)
+    print('Data Instance setup.')
+
+    x_rng = [args.xmin,args.xmax] 
+    y_rng = [args.ymin,args.ymax]
+    z_rng = [args.zmin,args.zmax]
+    #Verify that the cutout uploaded correctly.
+    attempts = 0
+    while attempts < 3:
+        try:
+            cutout_data = rmt.get_cutout(instance_setup, args.res, x_rng, y_rng, z_rng)
+            break
+        except HTTPError as e: 
+            if attempts < 3:
+                attempts += 1
+                print("Obtained HTTP error from server. Trial {}".format(attempts))
+            else:
+                print("Failed 3 times: {}".format(e))
+    # Data will be in Z,Y,X format
+    # Change to X,Y,Z for pipeline
+    cutout_data = np.transpose(cutout_data,(2,1,0))
+
+    def _upload(f):
```

Comment:
again, is it necessary to upload here? should s3wrap help with this? I think building this functionality into the tool directly risks a mismatch with the other workflows. consider removing